app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.java

```diff
@@ -480,6 +482,17 @@ public void onComplete() {
                         });
 
                 break;
+            case 96:
+                // TODO: why does it break with v4?
+                ncApi.setChatReadMarker(credentials,
+                                        ApiUtils.getUrlForSetChatReadMarker(1,
+                                                                            currentUser.getBaseUrl(),
+                                                                            conversation.getToken()),
+                                        conversation.lastMessage.jsonMessageId)
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .retry(1)
+                    .subscribe(genericOperationsObserver);
```

Comment:
Here a `break` is missing. Currently a click on _Mark as read_ also marks the conversation as favorite ⭐

Reply:
Fixed via https://github.com/nextcloud/talk-android/pull/1752/commits/0ba210a0cae90883395eb8aeec190a063831a567